dfm-sideline-sidekick-app/icons/circleIcon.tsx

```diff
@@ -0,0 +1,30 @@
+import { SearchIcon } from '../icons/searchIcon';
+import { StyleSheet, View } from 'react-native';
+import { Svg, Circle, G } from 'react-native-svg'; 
+
+const styles = StyleSheet.create({
```

Comment:
Not sure if it's necessary as it's such a small style sheet, but for consistency's sake, it might be good to move this to its own file as well.